website/static/js/dashboard.js

```diff
@@ -836,7 +853,7 @@ var Collections = {
                     'data-placement' : 'bottom'
                 }, ''),
                 m('.pull-right', [
-                    m('button.btn.btn-xs.btn-success[data-toggle="modal"][data-target="#addColl"]', m('i.fa.fa-plus')),
+                    !viewOnly ? m('button.btn.btn-xs.btn-success[data-toggle="modal"][data-target="#addColl"]', m('i.fa.fa-plus')) : '',
```

Comment:
In addition to not displaying the 'Add Collection' button, should there be a check in the `_addCollection` function to prevent it from being called on an Institution?